### PR TITLE
Changed README to contain basic description and link to main chat

### PR DESCRIPTION
**Pull request by Psyhich**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-Lol
+<h1><img align="center" src="img.png"></img>Nightly-IT</h1>
```
Comment (DenisLobur) [inline profile/README.md:1]: - align attribute is obsolete
- closing </img> tag is redundand, as  ia an empty tag that has no content
- missing alt attribute in img
- move img.png to assets folder (profile/assets/imgages/img.png)

Comment (Psyhich) [reply profile/README.md:1]: Align attribute is needed to center text around image

Comment (DenisLobur) [reply profile/README.md]: via CSS maybe?

Comment (DenisLobur) [reply profile/README.md]: or you can just embed attribute style, like <h1 style="text-align: center"> </h>

**Comment by DenisLobur** [comment]
LGTM. just merge

**Review by DenisLobur — APPROVED** [review]

**Pull request merged**